_posts/2020-07-20-municipal-boundaries-update-for-first-half-of-2020.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Municipal Boundaries Update for first half of 2020'
+author:
+  display_name: Mike Heagin
+  email: mheagin@utah.gov
+date: 2020-07-20 19:07:07
+categories:
+  - SGID Blog
+tags:
+  - Municipal Boundary
+  - municipalities
+  - boundaries
+  - annexations
+---
+
+As you may have noticed a lot of different things going on in the world out there. AGRC is now mainly working from our homes. And communication through the internet and phones. As we transitioned to a more telecommuting environment we had a few bumps in the road, nothing drastract, and our data was always available to our users. Apologies for not getting updates out there for the municipal boundaries in a while (6 months), but here we are now. Still been busy as business goes on and the wheels of progress keep turning. Growth may have slowed a little but not really a noticeable amount from an annexation point of view.
+
+We are at a total of 56 annexations so far this year which puts us in similar territory as the previous years. Again Utah County leads the way with Cache County following. This year several other counties have joined them. Iron, Sevier, and Tooele counties, with more change coming for Tooele, but I won’t mention those yet.
+   
+Hope everyone is staying safe and healthy. As always feel free to check out all the changes and make sure your boundaries are where you think they should be, view them on a the [Utah Municipality Modifications web map](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63). 
+To review the documents you can go the [Utah Lt. Governor Municipal Certifications website](https://municert.utah.gov/). Changes may be coming for this location, more on this later.
+
+Following are the changes and annexations since December 31, 2019 listed by County and City:
+
+| County | City | Number of Changes |
+| --- | --- | --- |
+| Cache | Hyrum | 1 |
+| | Logan | 2 |
+| | Nibley | 1 |
+| | Smithfield | 1 |
+| | Trenton | 1 |
+| Davis | Layton | 3 |
+| | Clearfield Syracuse adjustment | 1 |
+| Emery | Castle Cale | 1 |
+| Garfield | Panguitch  | 1 |
+| Iron | Cedar City  | 1 |
+| | Paragonah | 2 |
+| Salt Lake | Millcreek | 1 |
+| Sevier | Elsinore | 1 |
+| | Richfield | 3 |
+| | Sigurd | 1 |
+| SanJuan | Monticello | 2 |
+| Tooele | Grantsville | 2 |
+| | Tooele | 1 |
+| Utah | American Fork | 2 |
+| | Elk Ridge | 2 |
+| | Lehi | 9 |
+| | Lehi Highland adjustment | 1 |
+| | Lehi Saratoga Springs adjustment | 1 |
+| | Payson  | 1 |
+| | Pleasant Grove  | 1 |
+| | Spanish Fork  | 2 |
+| Wasatch | Charleston Town | 1 |
+| | Heber City  | 1 |
+| | Midway  | 1 |
+| Washington | Santa Clara  | 1 |
+| | St George  | 1 |
+| | Ivans Santa Clara adjustment  | 1 |
```

Comment:
```suggestion
| | Ivins-Santa Clara adjustment  | 1 |
```

Reply:
Also, with a quick Google search, it looks like this term more frequently comes up with Santa Clara-Ivins rather than Ivins-Santa Clara--but you might know better than I would on this.